personal/DariuszRatajczak/Reports/zajecia2.Rmd

```diff
@@ -0,0 +1,89 @@
+---
+title: "Zajecia2"
+author: "Dariusz Ratajczak"
+date: "4 04 2020"
+output: html_document
+---
+
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
+
+```
+
+```{r}
+
+library(sf)
```

Comment:
zalecam wrzucanie wszystkich library() do pierwszego bloku setup. Po to jest właśnie setup

Reply:
czy mam zatem poprawić ten plik i przerzucić te library do sekcji setup, czy zostawić ten plik ?